app/helpers/date_range_helper.rb

```diff
@@ -28,7 +28,13 @@ def this_year
   end
 
   def selected_interval
-    date_range_params.split(" - ").map { |d| Date.strptime(d, "%B %d, %Y") }
+    date_range_params.split(" - ").map do |d|
+      begin
```

Comment:
@dorner according to rubocop, this `begin` is redundant?

Reply:
Pushed a fix!